File: src/main/scala/jk_5/nailed/server/tileentity/OldStatEmitterConverter.scala
package jk_5.nailed.server.tileentity

import net.minecraft.nbt.NBTTagCompound
import net.minecraft.tileentity.TileEntity
import org.apache.logging.log4j.LogManager

/**
 * No description given
 *
 * @author jk-5
 */
class OldStatEmitterConverter extends TileEntity {

  val logger = LogManager.getLogger

  override def readFromNBT(tag: NBTTagCompound){
    super.readFromNBT(tag)
    logger.info(tag.toString)
  }

  override def writeToNBT(tag: NBTTagCompound){
    super.writeToNBT(tag)
  }
}

File: src/main/java/jk_5/nailed/server/tileentity/OldStatEmitterConverter.java
package jk_5.nailed.server.tileentity;

import net.minecraft.nbt.NBTTagCompound;
import net.minecraft.tileentity.TileEntity;
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

public class OldStatEmitterConverter extends TileEntity {

    private static final Logger logger = LogManager.getLogger();

    @Override
    public void readFromNBT(NBTTagCompound tag) {
        super.readFromNBT(tag);
        logger.info(tag.toString());
    }

    @Override
    public void writeToNBT(NBTTagCompound tag) {
        super.writeToNBT(tag);
    }
}
